docs/docusaurus/versioned_docs/version-0.17/changelog.md

```diff
@@ -3748,7 +3748,7 @@ title: Changelog
 ### 0.13.38
 * [FEATURE] Atomic Renderer: Initial framework and Prescriptive renderers ([#3529](https://github.com/great-expectations/great_expectations/pull/3529))
 * [FEATURE] Atomic Renderer: Diagnostic renderers ([#3534](https://github.com/great-expectations/great_expectations/pull/3534))
-* [BUGFIX] runtime_parameters: {batch_data: Spark DF} serialization ([#3502](https://github.com/great-expectations/great_expectations/pull/3502))
+* [BUGFIX] runtime_parameters: \{batch_data: Spark DF} serialization ([#3502](https://github.com/great-expectations/great_expectations/pull/3502))
```

Comment:
weird that we don't need to escape the close curly, but this does look correct in my markdown previewer 🤷

Reply:
@tyler-hoffman it is odd, but it is what is recommended. I tested the changes too.